src/pages/apply.js

```diff
@@ -0,0 +1,1169 @@
+import React, { useState } from "react"
+import { navigate } from "gatsby"
+import styled from "styled-components"
+import { useToasts } from "react-toast-notifications"
+import Select from "react-select"
+
+import {
+  Form,
+  Label,
+  Input,
+  Checkbox,
+  CheckboxInput,
+  TextArea,
+  Button,
+  Required,
+  Center,
+  H2,
+} from "../components/SharedStyledComponents"
+
+import { colorRed } from "../utils/styles"
+import validateEmail from "../utils/validateEmail"
+import isDev from "../utils/isDev"
+import * as styles from "../utils/styles"
+
+import { COUNTRY_OPTIONS, TIMEZONE_OPTIONS } from "../constants"
+
+const StyledForm = styled(Form)`
+  margin: 2rem auto;
+  max-width: ${styles.screenSizeXL};
+`
+
+const StyledSelect = styled(Select)`
+  margin-top: 0.5rem;
+`
+
+const StyledLabel = styled(Label)`
+  display: ${({ display = true }) => (display ? "flex" : "none")};
+  margin-bottom: 2.5rem;
+`
+
+const ErrorMessage = styled.small`
+  color: ${colorRed};
+`
+
+const ErrorDiv = styled.div`
+  margin-top: 0.5rem;
+  height: 0rem;
+  width: 100%;
+`
+
+const StyledButton = styled(Button)`
+  margin-bottom: 1rem;
+`
+
+const GENDER_TYPES = ["She/Her", "Him/His", "They/Them", "Other"]
+
+const YESNO = ["Yes", "No"]
+
+const PROJECTRESEARCHIDEA = [
+  "Yes, I have a project/research being implemented",
+  "Yes, I have a project idea but it is in its early stages",
+  "No, I have an idea, but the project/research is not currently being implemented",
+  "Other",
+]
+
+const REFERRALSOURCE = [
+  "Social media channels",
+  "Blog post",
+  "Through my network - someone shared the opportunity with me",
+  "Other (please specify)",
+]
+
+const requiredFields = [
+  "lastName",
+  "POCisAuthorisedSignatory",
+  "authorisedSignatoryInformation",
+  "contactEmail",
+  "phone",
+  "gender",
+  "country",
+  "timezone",
+  "title",
+  "isAffiliated",
+  "ethKnowledge",
+  "resumeLink",
+  "introVideoLink",
+  "projectResearchIdea",
+  "projectStatusOther",
+  "projectName",
+  "projectDescription",
+  "projectLeaderReasons",
+  "projectGoals",
+  "proposedTimeline",
+  "requestedAmount",
+  "problemBeingSolved",
+  "isYourProjectPublicGood",
+  "projectReasons",
+  "plansForBroaderCommunity",
+  "plansForScaling",
+  "repeatApplicant",
+  "referralSource",
+  "referralSourceIfOther",
+  "firstReferenceContact",
+  "secondReferenceContact",
+  "memeLink",
+  "memeDescription",
+]
+
+const emailFields = ["contactEmail"]
+
+const urlFields = [
+  "resumeLink",
+  "introVideoLink",
+  "projectPreviousWork",
+  "memeLink",
+]
+
+const RequiredError = () => <ErrorMessage>Field is required</ErrorMessage>
+const EmailError = () => (
+  <ErrorMessage>Please provide a valid email address</ErrorMessage>
+)
+const UrlError = () => <ErrorMessage>Ensure link is a valid URL</ErrorMessage>
+
+const DevconGrantsForm = () => {
```

Comment:
```suggestion
const ApplicationForm = () => {
```
Let's rename this component 😏